package.json

```diff
@@ -55,7 +55,7 @@
     "@material-ui/core": "3.9.4",
     "@testing-library/react": "10.4.9",
     "classnames": "2.2.6",
-    "cozy-bar": "7.9.0",
+    "cozy-bar": "7.13.3",
```

Comment:
According to the commit message `Removes a whole lot of dependencies` they are here some removing missing, or is it by upgrading cozy-bar from 7.9 to 7.13 ?

Reply:
Since cozy-bar has removed some of its dependencies, by upgrading it, we removed those dependency from here.